src/runtime_src/xdp/profile/plugin/device_offload/device_offload_plugin.cpp

```diff
@@ -93,9 +113,19 @@ namespace xdp {
 			     trace_buffer_size,            // trbuf_size,
 			     continuous_trace) ;           // start_thread
```

Comment:
should pass false here if calling read_trace_init. this code needs to be looked into in details.